bin/console

```diff
@@ -7,8 +7,8 @@
 
 set_time_limit(0);
 
-require_once __DIR__.'/bootstrap.php.cache';
-require_once __DIR__.'/AppKernel.php';
+require_once dirname(__DIR__).'/app/bootstrap.php.cache';
```

Comment:
As others have said, this lives in `/var` - and the same change will need to go in the front controllers.